packages/core/src/metrics/MetricsService.ts

```diff
@@ -19,18 +21,19 @@ export class MetricsService {
 
   protected publisher?: MetricsPublisher;
   protected configuration?: ServiceConfiguration;
-  private readonly defaultMetrics: Metrics[];
-
-  constructor(appConfig: AeroGearConfiguration) {
-    const configuration = new ConfigurationHelper(appConfig).getConfigByType(MetricsService.TYPE);
-    this.defaultMetrics = this.buildDefaultMetrics();
+  private readonly defaultMetrics?: Metrics[];
 
+  constructor() {
+    const configuration = coreInstance.getConfigByType(MetricsService.TYPE);
     if (configuration && configuration.length > 0) {
+      this.defaultMetrics = this.buildDefaultMetrics();
       this.configuration = configuration[0];
       this.publisher = new NetworkMetricsPublisher(this.configuration.url);
       this.sendInitialAppAndDeviceMetrics();
     } else {
-      console.warn("Metrics configuration is missing. Metrics will not be published to remote server.");
+      console.warn("Metrics configuration is missing." +
+        "Metrics will not be published to remote server.",
+        coreInstance);
```

Comment:
Not sure about printing coreInstance, it is for internal use. Maybe print just the configs?

Reply:
How I did not see that :)